### DISCUSSION TITLE
Print parameters profiles?

### DISCUSSION THREAD

**Discussion by robingantois** [sn4k3/UVtools#158] [Q&A] [answered]
Is there a possibility to export and import the print parameters? I cannot change some settings in prusaslicer, so I always have to enter manually in UVtools. It would be easy if I could just import certain settings as profiles. Or am I missing something?

**Comment by sn4k3** [comment, marked as answer]
Atm you can't have profiles for print parameters, but you can enter them all on prusa slicer: https://github.com/sn4k3/UVtools#custom-material-notes-and-printer-notes-keywords-for-prusaslicer

**Reply by robingantois** [reply]
Thank you for your answer. I had overlooked that!!!
Indeed i can manipulate all parameters with 'Printer Notes' Exept 'BottomLightOffDelay' nothing changes there and I keep seeing the same result in UVtools it indicates 8s.

**Reply by sn4k3** [reply]
What printer are you using?

**Reply by sn4k3** [reply]
Btw LightOffDelay's are auto set by UVtools if you are using the Automations (Settings - F12), I recommend you keep those on Auto

**Reply by robingantois** [reply]
Elegoo mars Pro

**Reply by robingantois** [reply]
I see that in the Automations settings the BottomLightOffDelay is set to 3sec if the value is 0. But I get 8!

**Reply by sn4k3** [reply]
The 3s is the extra time, so it will be 3s + the movement speed calculation, but i will confirm that in a minute

**Reply by sn4k3** [reply]
Ok i did a test with that printer, i set to 11 at prusaslicer, works for me. Make sure you dont have two BottomLightOffDelay lines, only the first get read

![image](https://user-images.githubusercontent.com/113281/109394822-ddadc780-7920-11eb-9adc-d69e7c3635fb.png)

**Reply by robingantois** [reply]
Thank you very much! I did try to get that parameter to 0, but now understand that is not a good idea!

**Reply by robingantois** [reply]
I tried to provide a profile of chitubox in prusaslicer. In chitubox, the bottom-delay-off is set to 0 by default

**Reply by sn4k3** [reply]
To get it to 0 you must disable the automation, or set it to 1, up to a value it will behave like 0 because that is the raw and firmware cap that value, which should never be like that due it require speed and height calculations to know the real value, chitubox should had that value just like i do on UVtools, you put the real extra time you want to wait. 3s on the automation will compute the right value for you, while on chitubox you must do it your self and that 3s would be something like 8 for example

**Reply by sn4k3** [reply]
On Tools - Calculator - Light Off Delay, you can find more information

**Reply by robingantois** [reply]
thank you for your time. Great piece of software, by the way

**Reply by sn4k3** [reply]
UVtools auto fixes some stuff on fly, some people thinks it's a bug ;)

**Reply by robingantois** [reply]
i think it is great!
